lib/anvil/utils.go

```diff
@@ -20,7 +21,7 @@ func FundAccounts(ctx context.Context, rpc string, amount *big.Int, accounts ...
 
 	for _, account := range accounts {
 		result := make(map[string]any)
-		err = client.Client().CallContext(ctx, &result, "anvil_setBalance", account, amount)
+		err = client.Client().CallContext(ctx, &result, "anvil_setBalance", account, hexutil.EncodeBig(amount))
```

Comment:
This was the fix. The rest is tests / adding `endpoint` to `anvil.Start(...)` return